modules/runner/src/testFixtures/java/org/apache/ignite/internal/Cluster.java

```diff
@@ -254,7 +260,7 @@ public CompletableFuture<IgniteImpl> startNodeAsync(int nodeIndex, String nodeBo
         String config = IgniteStringFormatter.format(
                 nodeBootstrapConfigTemplate,
                 BASE_PORT + nodeIndex,
-                CONNECT_NODE_ADDR,
```

Comment:
If we eliminate usage of this constant, do we want to remove it from this class as well?